_announcements/01_apple_common_removal.md

```diff
@@ -0,0 +1,319 @@
+7. When prompted, enter your device passcode.
+8. Click *Install* (top right), and *Install* again.
+9. Click *Done*.
+
+Next, you'll need to enable full trust for COMMON:
+
+1. From the iOS device's *Home* screen, go to *Settings* -> *General* -> *About* -> *Certificate Trust Settings*.
+2. Beneath *Enable Full Trust for Root Certificates*, toggle _ON_ for the Federal Common Policy CA entry. 
+3. When certificate appears, click *Continue*.
+4. You can now successfully navigate to an intranet site whose TLS certificate was issued by a Federal PKI CA.
+
+
+#### Option 2:&nbsp;&nbsp;Install COMMON Using Apple Configuration Profile
+You can use the macOS configuration profile procedures above to install COMMON on iOS devices: [Apple Configuration Profiles](#install-using-an-apple-configuration-profile).
+
+
+### tvOS&nbsp;&mdash;&nbsp;Install COMMON Options
+If you would like to install COMMON on tvOS government-furnished equipment, email us at fpki@gsa.gov.
+
+## Frequently Asked Questions
+
+### 1.&nbsp;&nbsp;Why is COMMON being removed? 
+The Federal PKI CAs don't comply with Apple's requirements for globally trusted TLS/SSL certificates. Apple's requirements include: 
+
+**a.&nbsp;&nbsp;Requirement for Fully-Qualified Domain Names (FQDNs)**<br>
+Apple plans to restrict TLS/SSL certificates to only those certificates using FQDNs ending in .gov, .mil, or fed.us. Some federal agencies issue TLS/SSL certificates to intranet assets. These certificates either:&nbsp;&nbsp;don't have FQDNs; contain intranet domains that don't end in .gov, .mil, or fed.us; or use short names (aliases). Under Apple's requirements, these agencies would need to reissue, re-install, and reconfigure all "non-compliant" certificates and applications. The Federal PKI community has determined that this would have a negative impact on mission applications on the intranets.
+
+**b.&nbsp;&nbsp;Requirement for public audit**<br>
+The Federal PKI follows a government auditing standard, and we have not restricted our issuance of TLS/SSL certificates to only the .gov and .mil domains. Under the requirements, all CAs in Federal PKI that could issue TLS/SSL certificates are required to submit a non-government audit or be technically constrained. Federal PKI has **not** technically constrained our CAs.
+
+**c.&nbsp;&nbsp;Requirement to disclose Certificate Practice Statements and Incident Post-Mortem Reports**<br>
+Public trust requires public disclosure and transparency. All Federal PKI CAs would be required to publicly post their Certificate Practice Statements and their Audit Letters. The Federal PKI community has attempted to disclose all Certificate Practice Statements for a number of years. However, some federal agencies include sensitive information in these documents and cannot disclose the documents publicly.
+
+**d.&nbsp;&nbsp;Requirement to create new issuing Certification Authorities (CAs)**<br>
```

Comment:
I'd remove? 

If we want to be specific here - the four items are: 

Certificate Transparency
Public disclosure and audits
Separating CAs by purpose
Symantec Distrust

Three of the four could be listed as informational?

Reply:
For now, I opted to remove. 

I'm struggling to think of a straight forward way of tying in CT/Symantec Distrust without really adding some weight/complexity to the page (e.g., to explain the CT influence well I feel like I'd need to reference root_stores.json and describe how user agents are tracked/pruned over time to highlight how removing now will save us pain in the future). 

What do you think?

Reply:
Makes sense.  I was thinking of the upcoming CT, not just the chrome implementation too.  You are correct IMO: let's keep it simpler first.  It's already long enough!